migrate/cmd.go

```diff
@@ -0,0 +1,83 @@
+package migrate
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/server"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+
+	"github.com/kava-labs/kava/migrate/v0_8"
+	v032tendermint "github.com/kava-labs/kava/migrate/v0_8/tendermint/v0_32"
+)
+
+const (
+	flagGenesisTime = "genesis-time"
+	flagChainID     = "chain-id"
+)
+
+// MigrateGenesisCmd returns a command to execute genesis state migration.
+func MigrateGenesisCmd(_ *server.Context, cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "migrate [genesis-file]",
+		Short:   "Migrate genesis from kava v0.3 to v0.8",
```

Comment:
So the main difference between kava and how the SDK does migration, is that the SDK supports multiple versions at a time and Kava supports only a single point-to-point migration?

Reply:
yeah. I started copying how the sdk does migrations, but I think migrating between multiple versions would almost never be used by us or others. That and it would add a big maintenance burden to make sure all the old types are still compatible after upgrading dependencies.